scripts/technical-debt-metrics.sh

```diff
@@ -84,6 +84,8 @@ titlesPathsAndRegexes=(
   "disabled simpNF lints"          "*"      "nolint simpNF"
   "erw"                            "*"      "erw \["
   "maxHeartBeats modifications"    ":^MathlibTest" "^ *set_option .*maxHeartbeats"
+  "cases'"                         "*"      " cases' "
+  "induction'"                     "*"      " induction' (?!\w+\s*:)"
```

Comment:
```suggestion
  "induction'"                     "*"      " induction' (?\!\w+\s*:)"
```
maybe

Reply:
With or without escaping `!`, running the script, I get
```bash
|0|0|induction'|
```

Reply:
(Running the script means both using `./scripts/technical-debt-metrics.sh` and actually typing out the `git grep` regex.

Reply:
I would be happy with the simpler reflex that simply excludes `:`.

If we want to do this we'll, maybe this should be reimplemented as a techDeptsLinter.

Reply:
I have reverted it.